src/lib/data/navigation.ts

```diff
@@ -119,8 +122,17 @@ export class NavigationContext {
 
     private updateHeadings() {
         const document = this.catalog.documents.find((b) => b.bookCode === this.book);
-        this.title = document.toc;
-        this.name = document.h;
+        if (document) {
+            this.title = document.toc;
+            this.name = document.h;
+            return;
```

Comment:
Are you still not going to find the html book when searching the catalog?  Not sure what this and the next one are doing

Reply:
The old code was pulling out the `title` and `name` properties from the document in the catalog. I tried to keep it consistent.  However, I don't see where these are even being used.